### DISCUSSION TITLE
第一次作業: actions (workflow)問題

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#70] [General]
![1](https://user-images.githubusercontent.com/91121500/136431218-bff60dbe-6c5a-4efd-97d2-38e2b18a3294.png)

我第一次作業做完後儲存完並push到repositories裡後, 再點選actions，找不到 Github Classroom workflow, 
跳出來的畫面為上圖, 不知要裝設哪一個workflow?

**Comment by jerry850313** [comment]
先去<code>完成作業
Action才有東西

**Reply by EricAgumonJou** [reply]
我作業已完成且成功push到git hub的repositories裡，接著按action還是出現這個畫面

**Reply by jerry850313** [reply]
我建議複製在VScode完成的code再貼上github資料夾裡的
試試看
